repos/system_upgrade/common/libraries/rhui.py

```diff
@@ -450,220 +431,6 @@ def mk_rhui_setup(clients=None, leapp_pkg='', mandatory_files=None, optional_fil
 }
 
 
-# DEPRECATED, use RHUI_SETUPS instead
-RHUI_CLOUD_MAP = {
```

Comment:
actually, I am not 100% sure we want to drop the whole map at this very moment. I do not remember know whether there is or not actually a small value in reports based on that. To make it more simple, I would separate this change into own commit. possibly, all changes regarding rhui actually.

or we can ask @MichalHe to be sure :) 👋  

sorry, my review capacity is getting dried today.